lighthouse-core/driver/index.js

```diff
@@ -100,15 +100,19 @@ class Driver {
     const loadData = {};
     let pass = Promise.resolve();
 
+    if (config.traceName) {
+      loadData[config.traceName] = {};
```

Comment:
as long as the trace names are unique, they won't overwrite (and it's equivalent to accumulating `Object.assign(tracingData, loadData)` with the `loadData` set up as you have it in this PR)